nifti2/nifti2_io.c

```diff
@@ -7077,7 +7078,7 @@ static int nifti_write_extensions(znzFile fp, nifti_image *nim)
    list = nim->ext_list;
    for ( c = 0; c < nim->num_ext; c++ ){
       size = (int)nifti_write_buffer(fp, &list->esize, sizeof(int));
-      ok = (size == (int)sizeof(int));
+      bool ok = (size == (int)sizeof(int));
```

Comment:
It would be preferable to stick with variable definitions at the block headers, sticking with the style of this code.